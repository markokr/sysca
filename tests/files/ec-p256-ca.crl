-----BEGIN X509 CRL-----
MIIBBjCBrgIBATAKBggqhkjOPQQDAjAQMQ4wDAYDVQQDDAVlY3JlcRcNMDUwMjAy
MDAwMDAwWhcNNDUwNTE1MDAwMDAwWjA8MDoCAQIXDTE5MDUxOTAwMDAwMFowJjAK
BgNVHRUEAwoBATAYBgNVHRgEERgPMjAxODEyMDEwMDAwMDBaoC8wLTAKBgNVHRQE
AwIBATAfBgNVHSMEGDAWgBQNYPRIQmcRx0Y3F24ZznJUcEMeXzAKBggqhkjOPQQD
AgNHADBEAiAoexWjU0h3f8lj+BlMH/+4tiFIqajZkLmZe4xRsCg2nQIgODn8koQw
RLPGncyssDk5OqHyUwJqlo06kws5YwYzrxQ=
-----END X509 CRL-----
