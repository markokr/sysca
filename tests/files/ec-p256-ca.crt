-----BEGIN CERTIFICATE-----
MIIBhzCCAS2gAwIBAgIUcApJS4MMa0KWDzxALMR4M3SD8y4wCgYIKoZIzj0EAwIw
EDEOMAwGA1UEAwwFZWNyZXEwIBcNMTAwMTAxMDAwMDAwWhgPMjA1MDAxMDEwMDAw
MDBaMBAxDjAMBgNVBAMMBWVjcmVxMFkwEwYHKoZIzj0CAQYIKoZIzj0DAQcDQgAE
wi1RnDlGvnIZmz0jOTgFukv9kxKUGY5Z1iKCoCrZ5J9rCPOd7YlEJ2dpxq1D6JkC
D/hgkIy8OvL/oS4A0lmbcKNjMGEwDwYDVR0TAQH/BAUwAwEB/zAOBgNVHQ8BAf8E
BAMCAQYwHQYDVR0OBBYEFA1g9EhCZxHHRjcXbhnOclRwQx5fMB8GA1UdIwQYMBaA
FA1g9EhCZxHHRjcXbhnOclRwQx5fMAoGCCqGSM49BAMCA0gAMEUCIGEYxD6CO7kU
kV8GSv5UDyF3Yk/VLAXj2r3YSq2gQO5hAiEA033ZwAz4nhmoeo+IE7wXDTwZBEmL
B/kGx4UjFQCAzUQ=
-----END CERTIFICATE-----
