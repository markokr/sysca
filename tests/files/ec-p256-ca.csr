-----BEGIN CERTIFICATE REQUEST-----
MIH+MIGkAgEAMBAxDjAMBgNVBAMMBWVjcmVxMFkwEwYHKoZIzj0CAQYIKoZIzj0D
AQcDQgAEwi1RnDlGvnIZmz0jOTgFukv9kxKUGY5Z1iKCoCrZ5J9rCPOd7YlEJ2dp
xq1D6JkCD/hgkIy8OvL/oS4A0lmbcKAyMDAGCSqGSIb3DQEJDjEjMCEwDwYDVR0T
AQH/BAUwAwEB/zAOBgNVHQ8BAf8EBAMCAQYwCgYIKoZIzj0EAwIDSQAwRgIhAMu2
/VxAbcR5VyFZLrUJV4jnRj4vahJme/LG8uBNQEGgAiEA70htyMNEgoYVGSJhCdzu
kp+yZqD/vWlo0gtxmDTATjs=
-----END CERTIFICATE REQUEST-----
