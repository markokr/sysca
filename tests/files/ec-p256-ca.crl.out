Issuer Name: CN = ecreq
Authority Key Identifier: 0d60f448426711c74637176e19ce725470431e5f
CRL Scope: all
CRL Number: 01
Last update: 2005-02-02 00:00:00
Next update: 2045-05-15 00:00:00
Revoked certificate:
  Serial: 02
  Revocation Date: 2019-05-19 00:00:00
  Invalidity Date: 2018-12-01 00:00:00
  Reason: key_compromise
  Issuer GNames: dn:/CN=ecreq/
